.gitmodules

```diff
@@ -10,3 +10,6 @@
 [submodule "libvast/aux/caf"]
 	path = libvast/aux/caf
 	url = https://github.com/actor-framework/actor-framework
+[submodule "contrib/vast-plugins"]
+	path = contrib/vast-plugins
+	url = git@github.com:tenzir/vast-plugins
```

Comment:
To check in CI if this is in the correct branch:

```
git -C contrib/vast-plugins fetch origin main
git -C contrib/vast-plugins merge-base --is-ancestor $(git -C contrib/vast-plugins rev-parse HEAD) $(git -C contrib/vast-plugins rev-parse origin/main)
```

The second command returns a non-zero exit code if the commit the submodule is pinned to is not in origin/main.

IIRC we wanted to add this as a separate job to the workflow that is required to pass the branch protections. A separate job so it prohibits you from merging, but doesn't stop the other jobs from running.